ddclient.in

```diff
@@ -4238,6 +4259,174 @@ sub nic_dnsexit_update {
     }
 }
 ######################################################################
+## nic_dnsexit2_examples
+######################################################################
+sub nic_dnsexit2_examples {
+    return <<"EoEXAMPLE";
+o 'dnsexit2'
+
+The 'dnsexit2' protocol is the new API protocol used by the dynamic hostname services
+of the 'DNSExit' dns services. This is currently used by the free
+dynamic DNS service offered by www.dnsexit.com.
+
+Configuration variables applicable to the 'dnsexit2' protocol are:
+  protocol=dnsexit2         ##
+  password=YourAPIKey       ## API Key of your account.
+  server=api.dnsexit.com    ## defaults to api.dnsexit.com.
+  path=/dns/                ## defaults to /dns/.
+  record-type=A             ## defaults to A record.
+  ttl=5                     ## defaults to 5 minutes.
+  min-error-interval=8m     ## defaults to 8 minutes.
```

Comment:
What does this do?

Reply:
I was not sure whether I had to add this here, I thought this would be an override to the 'service-common-defaults' which is supposedly used to throttle the client from updating when there's a lot of errors. Can you provide guidance on this (i.e. should I remove it)?